model/model.mustache

```diff
@@ -73,25 +74,30 @@ module.exports = {
     if (!{{name}}.rev) {
       return Promise.reject(new httpError('BAD_REQUEST', 'No rev provided'));
     }
-    const valid = schema.validate({{name}});
-    if (!valid) throw new httpError('UNPROCESSABLE_ENTITY');
-    return r.table('{{pluralName}}')
-    .get({{name}}.id)
-    .update(
-      rethinkdb.ifRevMatches({{name}}),
-      { returnChanges: true }
-    )
-    .run()
-    .then(rethinkdb.checkRevError)
-    .then(rethinkdb.firstChange);
+
+    return r.table('{{pluralName}}').get({{name}}.id).run()
+    .then(current => {
```

Comment:
Hmmm I didn't consider that, but not sure what the alternative is either. Jsen requires the entire model to be present to be able to validate it, and not just the changed fields. Yet rethinkdb supports the later. I guess the alternative would be to force passing the entire document anytime you want to make an update despite rethinkdb not requiring it.